src/App.js

```diff
@@ -1,6 +1,6 @@
-import "./App.css";
-import { Navbar, Footer, Aside } from "./component";
-import { Home, Product } from "./pages/index";
```

Comment:
Here, I think you can remove the `index` from the path like you are doing for the `components`.

Reply:
Yeah, its mistake I guess thanks for pointing that out😅